src/wp-admin/css/customize-widgets.css

```
@@ -308,73 +311,135 @@ body.adding-widget #customize-preview {
 }
 
 /* dashicons-smiley */
-#available-widgets [class*="easy"] .widget-title:before { content: "\f328" / ''; top: -4px; }
+#available-widgets [class*="easy"] .widget-title:before {
+	content: "\f328";
+	content: "\f328" / '';
+	top: -4px;
+}
 
 /* dashicons-star-filled */
 #available-widgets [class*="super"] .widget-title:before,
-#available-widgets [class*="like"] .widget-title:before { content: "\f155" / ''; top: -4px; }
+#available-widgets [class*="like"] .widget-title:before {
+	content: "\f155";
+	content: "\f155" / '';
+	top: -4px;
+}
 
 /* dashicons-wordpress */
-#available-widgets [class*="meta"] .widget-title:before { content: "\f120" / ''; }
+#available-widgets [class*="meta"] .widget-title:before {
+	content: "\f120";
+	content: "\f120" / '';
+}
 
 /* dashicons-archive */
-#available-widgets [class*="archives"] .widget-title:before { content: "\f480" / ''; top: -4px; }
+#available-widgets [class*="archives"] .widget-title:before {
+	content: "\f480";
+	content: "\f480" / '';
+	top: -4px;
+}
 
 /* dashicons-category */
-#available-widgets [class*="categor"] .widget-title:before { content: "\f318" / ''; top: -4px; }
+#available-widgets [class*="categor"] .widget-title:before {
+	content:"\f318";
+	content:"\f318" / '';
+	top: -4px; }
```

Minor edits:
- The closing `}` (curly brace/bracket) is not on its own line in this ruleset.
- Starting with the 'dashicons-category' ruleset, `content` rules lack a space after the colon in `customize-widgets.css`.

```suggestion
	content: "\f318";
	content: "\f318" / '';
	top: -4px;
}
```